sentry-core/src/performance.rs

```diff
@@ -154,6 +164,56 @@ impl TransactionContext {
     pub fn set_sampled(&mut self, sampled: impl Into<Option<bool>>) {
         self.sampled = sampled.into();
     }
+
+    /// Get the sampling decision for this Transaction.
+    pub fn sampled(&self) -> Option<bool> {
+        self.sampled
+    }
+
+    /// Get the name of this Transaction.
+    pub fn name(&self) -> &str {
+        &self.name
+    }
+
+    /// Get the operation of this Transaction.
+    pub fn operation(&self) -> &str {
+        &self.op
+    }
+
+    /// Get the custom context of this Transaction.
+    pub fn custom(&self) -> Option<&CustomTransactionContext> {
+        self.custom.as_ref()
+    }
+
+    /// Update the custom context of this Transaction.
+    ///
+    /// For simply adding a key, use the `set_custom_key` method.
```

Comment:
Is `set_custom_key` meant to be the `custom_insert` method below?

Reply:
Ah, yes. Will fix - I renamed it after I changed the signature to be in line with hashmap's `insert` (return previous value)